toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -74,11 +75,11 @@ public final void test(final String name, final String idProject, final boolean
 		DAOManager.saveTestPlan(testPlanTemplate);
 	}
 
-	public final void testAndStore(String apiKey, final ITestPlan project) throws Exception {
-		testAndStore(apiKey, project, false);
+	public final void testAndStore(String apiKey, final ITestPlan testplan) throws IOException, IllegalAccessException {
+		testAndStore(apiKey, testplan, false);
 	}
 
-	public final void testAndStore(String apiKey, final ITestPlan testPlan, final boolean useRemoteRepository) throws Exception {
+	public final void testAndStore(String apiKey, final ITestPlan testPlan, final boolean useRemoteRepository) throws IOException, IllegalAccessException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.lang.IllegalAccessException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)